student.rb

```diff
@@ -1,12 +1,17 @@
-require_relative './person'
+require './person'
 
-class Student
-  def initialize(age, classroom, name, parent_permision)
-    super(age, name, parent_permision)
+class Student < Person
+  def initialize(age, classroom, name, parent_permission)
```

Comment:
- [ ] Before creating a constructor for _Student_ class it's a good practice marked the _classroom_ object _ `classroom` with `attr_reader`, as we only intend to _access (get). this obeject.

Reply:
thank you for your feedback, the issue  has been resolved